### PR TITLE
Add labels to settings.yml

### LINKED ISSUE #12: Populate labels
Use the `.github/settings.yml` file to populate labels in this repo automatically.

### PR DESCRIPTION
**Pull request by suzubara**
## Description 

Add the labels we've decided on to `settings.yml` so they can be generated automatically.

Fixes #12 

## Review Notes

This PR has to be merged to take effect. Once it's merged, the labels in `settings.yml` should be added to this repo.

### CODE REVIEW

**Review by jbecker01 — DISMISSED** [review]
Looks good!

File: .github/settings.yml [hunk 1]
```diff
@@ -18,13 +18,130 @@ repository:
...
+    color: '#ffa500'
+  - name: 'type: feature'
+    color: '#ffb732'
+  - name: 'type: security / ATO'
+    color: '#ffc966'
+  - name: 'type: enhancement'
+    color: '#ffdb99'
+  - name: 'type: maintenance'
+    color: '#ffedcc'
+  - name: 'type: tech debt'
```
Comment (noahfirth) [inline .github/settings.yml:79]: May the odds be ever in our favor if we start finding a bunch of things that need this.

Comment (suzubara) [reply .github/settings.yml:79]: 😂  I am hoping we can avoid actually using this label, I imagine we may refine these once we get going

File: .github/settings.yml [hunk 2]
```diff
@@ -18,13 +18,130 @@ repository:
...
+  - name: good first issue
+    color: '#351c75'
+  - name: 'severity: 1'
+    color: '#ff4c4c'
+  - name: 'severity: 2'
+    color: '#ff7f7f'
+  - name: 'severity: 3'
+    color: '#ffb2b2'
+  - name: 'severity: 4'
+    color: '#ffe5e5'
```
Comment (noahfirth) [inline .github/settings.yml:48]: Did we have these defined somewhere? 100% possible I just missed it.

**Review by esacteksab — DISMISSED** [review]
Wow!

**Review by noahfirth — DISMISSED** [review]
Looks good; if we have the definitions of these somewhere, that would be super helpful to keep us all on the same page.

**Review by esacteksab — APPROVED** [review]

**Pull request merged**